install/kubernetes/README.md

```diff
@@ -235,6 +235,14 @@ Here are below the configuration properties of the MQTT support feature:
 | `features.async.mqtt` | `username` | **Optional**. The username to use for connecting to secured MQTT broker. Default to `microcks`.                                          |
 | `features.async.mqtt` | `password` | **Optional**. The password to use for connecting to secured MQTT broker. Default to `microcks`.                                          |
 
+#### NATS feature details
+
+Here are below the configuration properties of the NATS support feature:
+
+| Section               | Property   | Description                                                                                                                              |
+| --------------------- | ---------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| `features.async.nats` | `url`      | **Optional**. The URL of NATS broker (eg: `my-nats-broker.example.com:4222`). Default is undefined which means that feature is disabled. |
```

Comment:
Do you plan to manage authentication to NATS broker as well? Ideally that should be mandatory for release

Reply:
For the first PR I actually did not think about doing it, for the simple reason I feel the PR is large enough. I know NATS has a bunch of different authentication options, so it might need some discussion on how to handle some of it 😄 So it's just my personal preference to slowly add features and don't do a hail-mary one massive PR.

Reply:
Are there any specifics as to which features you see as a must-have for it to be merged?

Reply:
Totally agree on the approach of incremental PR! Authentication is a must have to pretend "Enterprise Ready" but I have no list on required options... I think it totally depends on what's commonly used on NATS side. We'll see ;-)